src/gui/search/searchsortmodel.cpp

```diff
@@ -47,8 +119,47 @@ bool SearchSortModel::lessThan(const QModelIndex &left, const QModelIndex &right
         if (Utils::String::naturalSort(vL.toString(), vR.toString(), res))
             return res;
 
-        return QSortFilterProxyModel::lessThan(left, right);
+        return base::lessThan(left, right);
+    }
+
+    return base::lessThan(left, right);
+}
+
+bool SearchSortModel::filterAcceptsRow(int source_row, const QModelIndex &source_parent) const
```

Comment:
>Or I can add `override` keyword.

You can do this if you will be easier. By the way, use `override` for overriden methods is a really good idea (not only here). It really can prevent some errors in contrast to redundant re-use of `virtual` keyword.